### PR TITLE
bulk_add catch exception to not discard all data

### PR DESCRIPTION
**Pull request by doomedraven**
the uniq thing which we should fix after this is unicode url regex support, i will add an issue to keep track of that

### CODE REVIEW

File: core/web/api/investigation.py [hunk 1]
```diff
@@ -99,14 +100,18 @@ def bulk_add(self, id):
...
+                try:
+                    if node['type'] in globals() and issubclass(
+                            globals()[node['type']], Observable):
+                        _type = globals()[node['type']]
+
+                    n = _type.get_or_create(value=node['value'])
+                    if node['new_tags']:
+                        n.tag(node['new_tags'].split(', '))
+                    nodes.append(n)
+                except Exception as e:
```
Comment (tomchop) [inline core/web/api/investigation.py]: It would be good to catch a more specific type of exception here. Are you looking for `ObservableValidationError`?

**Comment by doomedraven** [comment]
Oh thanks for the point, will update once i will be at home

File: core/web/api/investigation.py (hunk 1 above)
Comment (doomedraven) [reply core/web/api/investigation.py]: thanks for point for better solution, updated

**Comment by doomedraven** [comment]
should be fine now no=

**Pull request merged**